pkgs/development/libraries/libproxy/default.nix

```diff
@@ -12,22 +12,47 @@ stdenv.mkDerivation rec {
     sha256 = "10swd3x576pinx33iwsbd4h15fbh2snmfxzcmab4c56nb08qlbrs";
   };
 
+  patches = lib.optionals stdenv.isDarwin [
+    (fetchpatch {
+      url = https://raw.githubusercontent.com/macports/macports-ports/master/net/libproxy/files/patch-libproxy-cmake.diff;
```

Comment:
Actually, do we need every one of these patches? It is not like we are building Qt4 or Perl bindings. It would be nice if you could add a comment with an error produced without each patch.